ivy/functional/backends/tensorflow/__init__.py

```diff
@@ -148,6 +150,45 @@
 
 native_inplace_support = False
 
+backend = "tensorflow"
+
+
+def _handle_inplace_mode():
```

Comment:
Given that throwing the warning in the `__init__.py` is leading to code duplication, let's just generate the warning in the backend handler instead as you were doing before, with a warning generated from [`set_backend`](https://github.com/unifyai/ivy/blob/8ffd9ffad6f715a3b0bdedb3bbafe52d0085d2e6/ivy/utils/backend/handler.py#L396), [`previous_backend`](https://github.com/unifyai/ivy/blob/8ffd9ffad6f715a3b0bdedb3bbafe52d0085d2e6/ivy/utils/backend/handler.py#L526) and [`with_backend`](https://github.com/unifyai/ivy/blob/8ffd9ffad6f715a3b0bdedb3bbafe52d0085d2e6/ivy/utils/backend/handler.py#L623) with this helper being moved to [`exceptions.py`](https://github.com/unifyai/ivy/blob/8ffd9ffad6f715a3b0bdedb3bbafe52d0085d2e6/ivy/utils/exceptions.py).